src/generalSettings/index.ts

```diff
@@ -12,6 +13,12 @@ export default class GeneralSettings extends BaseApi {
     return responseHandler.requiredList(response);
   }
 
+  public readRaw(requestParams?: RequestParams<GeneralSetting> | Object): Promise<ApiResponseOnSuccess<GeneralSetting[]>> {
+    const params = requestParams instanceof RequestParams ? requestParams.getParams() : requestParams;
+    const response = this._get<GeneralSetting>(`${this.getResourceName()}/read`, { params });
```

Comment:
```suggestion
    const response = this._get<GeneralSetting[]>(`${this.getResourceName()}/read`, { params });
```
This shouldn't compile without the array, but with the messed up types we have now it does. Should be fixed with #33 . I hope I'll have the time to get that PR ready in the next days

Reply:
fixed now only for readRaw, other cases will be fixed in #33